ch.tigerjython.TigerJython2.metainfo.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>ch.tigerjython.TigerJython2</id>
+
+  <name>TigerJython</name>
+  <summary>TigerJython IDE</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>custom</project_license>
+
+  <recommends>
+    <display_length compare="ge">482</display_length>
+  </recommends>
+
+  <description>
+    <p>
+      TigerJython is a free development environment for the Python programming language. If you want to learn programming, this is the tool for you. It already includes everything you need to start programming right away.
+    </p>
+    <p>
+      For all versions we guarantee not to change or spy your software.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">tigerjython.desktop</launchable>
+</component>
```

Comment:
This won't pass validation, make sure to validate https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/#use-flathubs-appstream-util

Reply:
The Licence we give is:

 > Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
 > * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
 >  * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
 > * Neither the name of the author nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
 > 
 > THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
How do I set the project_license?

Reply:
this looks ike BSD 3 clause. https://opensource.org/license/BSD-3-clause/

Reply:
`BSD-3-Clause`